### ISSUE TITLE
React Native Support?

### ISSUE THREAD

**Issue by ImAbhishekTomar** [nextauthjs/next-auth#2477]
### Question 💬

Possible to use this with React Native? If Yes; anyone please share reference. 

### How to reproduce ☕️

Possible to use this with React Native? If Yes; anyone please share reference. 

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
See #2294. Not sure what React Native would need, but #2294 is the right discussion anyway.

**Comment by jonlambert** [comment]
@balazsorban44 Totally understood that this has a dependency on #2294, but IMO it seems worthwhile to have a ticket to track native, non-browser support in addition.

I arrived here from Google and as a newcomer, it's not obvious what would need to be done to reach that once the work associated with #2294 is merged.

**Comment by balazsorban44** [comment]
I am fine with opening a separate issue, but that should be a proper feature request with someone doing the research on what is needed, etc. Since this was marked as a question, I was implied to point the OP to the most correct discussion at the time.

Keep in mind that this is not in my short-term goals for the library, so I won't be spending time working on it for the foreseeable future. What I would like is through #2294, to create engagement amongst other framework developers to use `next-auth` as their base.

If you would like to champion such an RFC and could actively helping out in the discussion, I welcome it! :green_heart:

**Comment by redbar0n** [comment]
Could this PR resolve this issue?

 feat: next-auth/expo https://github.com/nextauthjs/next-auth/pull/5240

**Comment by redbar0n** [comment]
Related discussion: https://github.com/nextauthjs/next-auth/discussions/3942#discussioncomment-2247534
